config/settings.yml

```diff
@@ -5,7 +5,7 @@ vsp_environment:
 virtual_hosts: ["127.0.0.1", "localhost"] # Safe host names
 
 # For CORS requests; separate multiple origins with a comma
-web_origin: http://localhost:3000,http://localhost:3001,http://127.0.0.1:3000,http://127.0.0.1:3001,null
+web_origin: http://localhost:3000,http://localhost:3001,http://127.0.0.1:3000,http://127.0.0.1:3001,https://department-of-veterans-affairs.github.io,null
```

Comment:
Hey @timwright12, I think you will need to update this setting in the devops repo. The settings.yml gets overwritten on the deploy with the settings here for each environment: [prod](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/prod-settings.local.yml.j2), [staging](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/staging-settings.local.yml.j2), [sandbox](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/sandbox-settings.local.yml.j2), [dev](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/dev-settings.local.yml.j2)

Reply:
@LindseySaari awesome thank you! Do I need to merge this PR and one in devops or just devops?

Reply:
I believe just devops unless you need that change for local development (which it doesn't appear that you do)

Reply:
excellent, thank you. I'll close this out